api/checks/summaries/compile_test.go

```diff
@@ -16,24 +16,40 @@ import (
 	"github.com/web-platform-tests/wpt.fyi/shared"
 )
 
-// To output the rendered content during execution of the test(s), set this flag,
-// e.g. go test ./api/checks/summaries -tags="small" -print_output -test.v
+// To output the rendered content during execution of the test(s), set this flag, e.g.
+// go test ./api/checks/summaries -tags="medium" -print_output -test.v
 var renderOutputToConsole = flag.Bool("print_output", false, "Whether to render compiled markdown during test execution.")
 
 func TestGetSummary_Completed(t *testing.T) {
-	foo := Completed{
-		DiffURL:  "foo.com/diff?before=chrome[master]&after=chrome@0123456789",
-		HostName: "foo.com",
-		HostURL:  "foo.com/results/",
-		SHAURL:   "foo.com/sha/",
+	master := shared.TestRun{}
+	master.BrowserName = "chrome"
+	master.Revision = "abcdef0123"
+	master.FullRevisionHash = strings.Repeat(master.Revision, 4)
+	pr := shared.TestRun{}
+	pr.BrowserName = "chrome"
+	pr.Revision = "0123456789"
+	pr.FullRevisionHash = strings.Repeat(pr.Revision, 4)
+	foo := Completed{}
+	foo.MasterRun = master
+	foo.PRRun = pr
+	foo.DiffURL = "foo.com/diff?before=chrome[master]&after=chrome@0123456789"
+	foo.HostName = "foo.com"
+	foo.HostURL = "foo.com/results/"
```

Comment:
nit: It looks like production code would include the scheme here. Consider including in tests for consistency.

Reply:
Done.